docs/features.md

```diff
@@ -7,19 +7,58 @@ autoscaling group, fck-nat can be configured to always attach a specific ENI at
 a consistent internal-facing IP address. Additionally, it is also possible to configure an already allocated EIP address
 that would be carried through instance refreshs.
 
-To enable these features, you'll need to create a config file at `/etc/fck-nat.conf` like this:
+Those features are controlled by `eni_id` and `eip_id` directive in the configuration file.
 
-```
-eni_id=<YOUR_ENI_ID>
-eip_id=<YOUR_EIP_ALLOCATION_ID>
-```
+## Metrics
 
-Once the fck-nat configuration is created, be sure to restart the service by running `service fck-nat restart`.
+One of the objectives of fck-nat is to offer as close as possible metric parity with Managed NAT Gateway. While the
+project supports various metrics similar to the managed NAT Gateway via Cloudwatch agent, each provider is responsible
+for passing their configuration to the agent via fck-nat's `cwagent_enabled`, and `cwagent_cfg_param_arn` directives
+within its configuration file.
 
-In the official fck-nat CDK construct, we configure this via UserData on the autoscaling group.
+As an example, you might use the following configuration file which have Cloudwatch agent report most of metrics
+provided in the managed NAT Gateway:
 
-## Metrics
+``` json
+{
+  "agent": {
```

Comment:
I'd recommend adding conntrack metrics to this based on our discussion around security groups: https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Agent-network-performance.html

Reply:
Initially meant to do it in two times, but it actually wasn't that much more work. I added a few more metrics from `ethtool` which seem just as relevant.

Furthermore, I added a little warning https://github.com/AndrewGuenther/fck-nat/pull/45/commits/3433913b6050c031ac7d4e0d46aa8866e7d1e0fd#diff-aff15ced1fe37506f41f2e1d93a4d8312529259f7803f8634ffb5eac4de47823R75-R76, as this Cloudwatch configuration adds up a lot in terms of cost, negating a good part of the gain from fixed costs in NAT Gateway.
Using Cloudwatch agent would therefore be more interesting to have on a more "serious" usage of fck-nat, or if the main driving cost is the data flowing through the NAT instance.

Reply:
Goddamn I always forget how pricey custom CloudWatch metrics are...

Appreciate the additional disclaimer!